tests/TraceContext/W3CTestService/symfony-setup

```diff
@@ -8,25 +8,28 @@ curl -sS https://get.symfony.com/cli/installer | bash
 mv /root/.symfony/bin/symfony /usr/local/bin/symfony
 
 # Set up Symfony test application
-cd tests/TraceContext/W3CTestService
+cd tests/TraceContext/W3CTestService || exit
```

Comment:
Not sure of the reasoning behind `|| exit` for these - I'd say either give a detailed error message or don't do this.